packages/cubejs-client-core/src/index.js

```diff
@@ -25,11 +25,13 @@ class CubejsApi {
     options = options || {};
     this.apiToken = apiToken;
     this.apiUrl = options.apiUrl || API_URL;
+    this.method = options.method || 'GET';
```

Comment:
In this case, it will always be `GET` by default

Reply:
oops forgot to change this file, I've updated it again. Thanks